Cargo.toml

```diff
@@ -44,7 +44,8 @@ zeroize = "1.5"
 # Optional ones
 js-sys = { version = "0.3", optional = true }
 lazy_static = { version = "1.4", optional = true }
-pyo3 = { version = "0.16.3", features = ["extension-module"], optional = true }
+# Build python package for version 3.7 and above: https://pyo3.rs/v0.17.3/features.html
```

Comment:
no specific comment for this line but rather on the file Cargo.toml:
- we must bump the version to 8.0.0 du to:
```
Description:
A publicly-visible struct cannot be imported by its prior path. A `pub use` may have been removed, or the struct itself may have been renamed or removed entirely.
        ref: https://doc.rust-lang.org/cargo/reference/semver.html#item-remove
       impl: https://github.com/obi1kenobi/cargo-semver-check/tree/v0.12.1/src/queries/struct_missing.ron

Failed in:
  struct cosmian_cover_crypt::api::ClearTextHeader, previously in file /home/manu/.cargo/registry/src/github.com-1ecc6299db9ec823/cosmian_cover_crypt-7.0.1/src/api.rs:339
  struct cosmian_cover_crypt::ClearTextHeader, previously in file /home/manu/.cargo/registry/src/github.com-1ecc6299db9ec823/cosmian_cover_crypt-7.0.1/src/api.rs:339
```
- why do we use the `develop` branch of `cosmian_crypto_core`? (crate publishing will fail outside a version reference)
- `criterion` version can be upgraded to `0.4`
- we could do some reordering as follow (if it is OK for you):
```
[package]
name = "cosmian_cover_crypt"
version = "8.0.0"
authors = [
  "Théophile Brezot <theophile.brezot@cosmian.com>",
  "Bruno Grieder <bruno.grieder@cosmian.com>",
]
documentation = "https://docs.rs/cosmian_cover_crypt/"
edition = "2021"
license = "MIT/Apache-2.0"
repository = "https://github.com/Cosmian/cover_crypt"
description = "Key Policy attribute encryption based on subset cover"

[lib]
crate-type = ["rlib", "cdylib", "staticlib"]
name = "cosmian_cover_crypt"
# The cdylib is only interesting if the `--features ffi` flag is set on build
# This does not seem to be actionable conditionally https://github.com/rust-lang/cargo/issues/4881

[[bench]]
harness = false
name = "benches"

[profile.bench]
debug = true

[features]
ffi = ["lazy_static"]
full_bench = []
python = ["pyo3"]
wasm_bindgen = ["js-sys", "wasm-bindgen"]

[dependencies]
abe_policy = "1.0"
cosmian_crypto_core = { git = "https://github.com/Cosmian/crypto_core", branch = "develop" }
hex = "0.4"
js-sys = { version = "0.3", optional = true }
lazy_static = { version = "1.4", optional = true }
leb128 = "0.2"
pyo3 = { version = "0.17.3", features = ["extension-module", "abi3", "abi3-py37"], optional = true }
rand_core = { version = "0.6", features = ["getrandom"] }
serde = { version = "1.0", features = ["derive"] }
serde_json = "1.0"
sha3 = "0.10"
thiserror = "1.0"
wasm-bindgen = { version = "0.2", features = ["serde-serialize"], optional = true }
zeroize = "1.5"

[dev-dependencies]
criterion = { version = "0.4", features = ["html_reports"], default_features = false }
wasm-bindgen-test = "0.3"

```

Reply:
I think that the current version of crypto core doesn't have the right serialization implementation, @tbrezot ?